tools/gh_api.py

```diff
@@ -276,8 +276,7 @@ def post_download(project, filename, name=None, description=""):
     """Upload a file to the GitHub downloads area"""
     if name is None:
         name = Path(filename).name
-    with open(filename, 'rb') as f:
-        filedata = f.read()
+    filedata = Path(filename).read_bytes()
```

Comment:
You could improve this method further by making sure it creates `Path` only once - currently it does twice.